src/i18n/es/index.js

```diff
@@ -0,0 +1,156 @@
+import potato from './potato';
+import print from './print';
+import tester from './tester';
+export default {
+  en: {
```

Comment:
wrong key. please use `es`, english is now broken 😄

Reply:
Oops, completely missed it! Got it now